examples/governor-bravo/BRAVO_01.sol

```diff
@@ -26,15 +26,11 @@ contract BRAVO_01 is GovernorBravoProposal {
 
     /// @notice Deploys a vault contract and an ERC20 token contract.
     function _deploy() internal override {
-        if (!addresses.isAddressSet("VAULT")) {
-            Vault timelockVault = new Vault();
-            addresses.addAddress("VAULT", address(timelockVault), true);
-        }
-
-        if (!addresses.isAddressSet("TOKEN_1")) {
-            MockToken token = new MockToken();
-            addresses.addAddress("TOKEN_1", address(token), true);
-        }
+        Vault timelockVault = new Vault();
+        addresses.addOrChangeAddress("VAULT", address(timelockVault), true);
+
+        MockToken token = new MockToken();
+        addresses.addOrChangeAddress("TOKEN_1", address(token), true);
```

Comment:
I added the reason in the PR description
> Change _deploy functions to add or change addresses. This is needed since we now use a Sepolia fork to run integration tests. We still want to be able to redeploy the integration test proposals contracts that exist on sepolia

Reply:
For running integration tests in a fork simulation, we still want to deploy the proposal contracts even if they have already been deployed. Otherwise, what is the point of having integration tests if they always run against an already executed proposal and aren't running the proposal?